src/web/components/menu/__tests__/Menu.test.jsx

```diff
@@ -0,0 +1,183 @@
+/* SPDX-FileCopyrightText: 2025 Greenbone AG
+ *
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+
+import {afterEach, describe, expect, test, testing} from '@gsa/testing';
+import React from 'react';
+import {rendererWith, screen} from 'web/utils/Testing';
+import Menu from 'web/components/menu/Menu';
+
+testing.mock('web/hooks/useTranslation', () => ({
+  default: () => [key => key],
+}));
+
+afterEach(() => {
+  testing.clearAllMocks();
+});
+
+const renderMenuWith = async ({capabilities, gmpSettings}) => {
```

Comment:
I don't think render is async and therefore renderMenuWith needs to be async.
```suggestion
const renderMenuWith = ({capabilities, gmpSettings}) => {
```